root.sh

```diff
@@ -181,6 +181,13 @@ JOBS=$((${JOBS:-1}*3/5))
 [[ $JOBS -gt 0 ]] || JOBS=1
 cmake --build . --target install ${JOBS:+-- -j$JOBS}
 
+if test -f "$HOME/.rootrc" && $(grep -q PluginPath "$HOME/.rootrc"); then
```

Comment:
No one reads those warnings and they will not be printed for people who download the binaries. So we need to find a better solution.